README.md

```diff
@@ -109,6 +110,19 @@ Hurray! You successfully made a contribution!
 
 ## <ins>Hacktoberfest community<ins>
 
+### **Contents**
+
+|  |         |      |      |     |
+| ---------| ------- | ---- | ---- | --- |
```

Comment:
Remove this empty row

Reply:
@adityaraute If I remove the top row, it will break the table, but if I fill it with the title, it makes it uneven like this:
![image](https://user-images.githubusercontent.com/70807500/132574232-9b0cf12b-3f85-4daf-b3b3-dd382e3fa820.png)
thanks.